page/login.js

```diff
@@ -8,6 +8,8 @@ function page (req, res) {
   var fs = require('fs')
   fs.readFile('html/login.html', 'utf8', function (err, html) {
     if (err) throw err
-    res.end(require('../src/html_creator').create_html(html, { title: 'Login', js: ['login'] }))
+
+    res.end(require('../src/html_creator').create_html(html, { title: 'Login', js: ['login', 'cookie_parser'] }))
```

Comment:
No js file with the name `cookie_parser` exists, this results in a 404.

Reply:
Fixed by latest commit